src/Metric.php

```diff
@@ -498,21 +522,46 @@ public function getPDFLinks(\DomXpath $xpath)
 
         foreach ($nodes as $node) {
             $href = $node->getAttribute('href');
-            
+
             if (strtolower(substr($href, -4)) == '.pdf') {
                 $links[] = array(
                     'value_found' => $href,
                     'context' => htmlspecialchars($xpath->document->saveHTML($node))
                 );
             }
         }
-        
+
+        return $links;
+    }
+
+    /**
+     * Get a array of Box.com links
+     *
+     * @param \DomXpath $xpath
+     * @return array - an array of links.  Each link is an associative array with 'href' and 'html' values
+     */
+    public function getBoxLinks(\DomXpath $xpath)
+    {
+        $links = array();
+        $nodes = $xpath->query("//xhtml:a");
+
+        foreach ($nodes as $node) {
+            $href = $node->getAttribute('href');
+
+            if (strpos($href, 'box.com') !== false) {
```

Comment:
Maybe do '.box.com' to not hit false positives like dropbox.com or sharebox.com

Reply:
Or actually, '.box.com' or '//box.com'

Reply:
Good catch, updated.